app/controllers/api/functions.php

```diff
@@ -865,3 +1053,141 @@
 
         $response->dynamic($execution, Response::MODEL_EXECUTION);
     });
+
+App::get('/v1/builds')
```

Comment:
Shouldn't build be nested under their tag? They should probably also use the function read and write permissions and not the execution ones. (`execution.read` -> `functions.read` and `execution.write` -> `functions.write`)

Reply:
Builds are not currently nested in their tag since builds need to be seperate from functions so static websites can use them aswell.